src/lib.rs

```diff
@@ -409,9 +427,21 @@ impl<'a, Key: Ord + Clone, Value> Entry<'a, Key, Value> {
 #[cfg(test)]
 mod test {
     use rand;
-    use std::thread;
     use std::time::Duration;
 
+    #[cfg(feature = "fake_clock")]
+    fn sleep(time: Duration) {
+        use fake_clock::FakeClock;
+        // adding one millisecond to simulate waiting a tiny bit longer than requested
```

Comment:
I'd put that in the test itself where necessary, rather than hide it in this method, so that it is clear from the test whether timestamps need to be strictly greater or just greater-or-equal.